client/src/components/HistoryViewer/HistoryViewer.scss

```diff
@@ -44,8 +48,13 @@
   white-space: nowrap;
 }
 
-// Bring any CMS tabs in the detail view closer to the table above it
 .history-viewer__version-detail {
+  &.history-viewer__version-detail--compare {
+    margin-left: $form-check-input-gutter;
+    margin-right: $form-check-input-gutter;
```

Comment:
I have a feeling that `$grid-gutter-width` might be better (assuming its the same computed value) since we use it more often in the admin variables sheet